extensions/amc-score-calculator/src/eight.tsx

```diff
@@ -0,0 +1,5 @@
+import Generic from './generic';
+
+export default () => {
+  return <Generic title={'AMC 8 Score Calculator'} correctValue={1} emptyValue={0} />;
```

Comment:
So does AMC 8 always returns the same number as number of correct answers? What's the value in this calculation? Or am I missing something?